### PR TITLE
Fix #672: Start JMX remote monitoring from the Diagnostic Port

### LINKED ISSUE #672: Start JMX remote monitoring from the Diagnostic Port
I would add the command to the DSO MBean

Hint:

https://stackoverflow.com/questions/27508185/is-it-possible-to-enable-remote-jmx-monitoring-programmatically

### PR DESCRIPTION
**Pull request by ramsai1729**

### CODE REVIEW

File: dso-l2/src/main/java/com/tc/stats/DSO.java [hunk 1]
```diff
@@ -66,6 +70,7 @@
 public class DSO extends AbstractNotifyingMBean implements DSOMBean {
 
   private final static Logger logger = LoggerFactory.getLogger(DSO.class);
+  private final static int DEFAULT_JMX_REMOTE_PORT = 5050;
```
Comment (myronkscott) [inline dso-l2/src/main/java/com/tc/stats/DSO.java]: Probably need this to be configurable somehow

Comment (ramsai1729) [inline dso-l2/src/main/java/com/tc/stats/DSO.java]: yeah, is there a way to pass this port info from Diagnostics entity?

Comment (myronkscott) [inline dso-l2/src/main/java/com/tc/stats/DSO.java]: I'd use the setter/getter bean pattern on the MBean and use Diagnostics.set

**Comment by myronkscott** [comment]
Can you add a test?  A unit test would suffice if JUnit will let you do it.

File: dso-l2/src/main/java/com/tc/stats/DSO.java (hunk 1 above)
Comment (myronkscott) [inline dso-l2/src/main/java/com/tc/stats/DSO.java]: you could even just start the server from the set method

**Pull request merged**